src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlSearchParameters.cs

```diff
@@ -0,0 +1,20 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Health.Fhir.Core.Models;
+using Microsoft.Health.Fhir.ValueSets;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Search
+{
+    public static class SqlSearchParameters
+    {
+        public const string ResourceSurrogateIdParameterName = "_resourceSurrogateId";
+
+        public static readonly Uri ResourceSurrogateIdUri = new Uri("http://hl7.org/fhir/SearchParameter/Resource-surrogateId");
```

Comment:
nit: Since this is of our own creation, should we make it in a Uri that's unique to us?

Reply:
Makes sense